html/elements/menu.json

```diff
@@ -93,9 +91,9 @@
               }
             },
             "status": {
-              "experimental": true,
-              "standard_track": true,
-              "deprecated": false
+              "experimental": false,
+              "standard_track": false,
+              "deprecated": true
```

Comment:
This feature is all-false, deprecated, and non-standard. Should this be deleted instead?

Reply:
Yes, the removal is up for review at https://github.com/mdn/browser-compat-data/pull/6955. One of the PRs will have to be rebased depending on which lands first. (I'd prefer to not do the removal here.)

Reply:
Great. Happy to let the removals happen elsewhere.